---
title: SvgConversionException
second_title: Aspose.Font for Java API Reference
description: Represents Font conversion exception for SVG format.
type: docs
weight: 69
url: /java/com.aspose.font/svgconversionexception/
---
**Inheritance:**
java.lang.Object, java.lang.Throwable, java.lang.Exception, java.lang.RuntimeException, java.lang.IllegalStateException, [com.aspose.font.FontException](../../com.aspose.font/fontexception)
```
public class SvgConversionException extends FontException
```

Represents Font conversion exception for SVG format. The exception can be thrown in case of errors during font conversion process.
## Constructors

| Constructor | Description |
| --- | --- |
| [SvgConversionException()](#SvgConversionException--) | Initializes new  SvgConversionException  object. |
| [SvgConversionException(String message)](#SvgConversionException-java.lang.String-) | Initializes new  SvgConversionException  object. |
| [SvgConversionException(String message, RuntimeException innerException)](#SvgConversionException-java.lang.String-java.lang.RuntimeException-) | Initializes new  SvgConversionException  object. |
### SvgConversionException() {#SvgConversionException--}
```
public SvgConversionException()
```


Initializes new  SvgConversionException  object.

### SvgConversionException(String message) {#SvgConversionException-java.lang.String-}
```
public SvgConversionException(String message)
```


Initializes new  SvgConversionException  object.

**Parameters:**
| Parameter | Type | Description |
| --- | --- | --- |
| message | java.lang.String | A message that describes the error. |

### SvgConversionException(String message, RuntimeException innerException) {#SvgConversionException-java.lang.String-java.lang.RuntimeException-}
```
public SvgConversionException(String message, RuntimeException innerException)
```


Initializes new  SvgConversionException  object.

**Parameters:**
| Parameter | Type | Description |
| --- | --- | --- |
| message | java.lang.String | A message that describes the error. |
| innerException | java.lang.RuntimeException | The exception that is the cause of the current exception. |

